docs/docs/03-syntax-and-usage/12-script-templates.md

```diff
@@ -188,6 +188,42 @@ The data in the script tag can then be accessed from client-side JavaScript.
 const data = JSON.parse(document.getElementById('id').textContent);
 ```
 
+Alternatively, you can add a helper that defines a constant, making it accessible in JavaScript scripts as a `const` variable.
+
+```templ
+// helper/helper.go
+package helper
+
+import (
+	"fmt"
+	"github.com/a-h/templ"
+)
+
+func AddScriptVar(k string, v string) templ.Component {
+	raw_script := fmt.Sprintf(`<script>const %s="%s"</script>`, k, v)
```

Comment:
If the key `k` or value `v` contains a quote `"`, the page could be subjected to a cross site-scripting attack.